hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/service/MonitorService.java

```diff
@@ -909,8 +911,13 @@ private void waitForHawkularServer() throws Exception {
                 if (response.code() != 200) {
                     log.debugf("Hawkular Metrics is not ready yet: %d/%s", response.code(), response.message());
                 } else {
-                    log.debugf("Hawkular Metrics is ready: %s", response.body().string());
-                    break;
+                    String bodyString = response.body().string();
+                    if (checkReallyUp(bodyString)) {
+                        log.debugf("Hawkular Metrics is ready %s", bodyString);
+                        break;
+                    } else {
+                        log.debugf("Hawkular Metrics is still starting", bodyString);
```

Comment:
Sorry @pilhuhn  - I just noticed this when I was about to press the merge button. Notice this debugf call passes in "bodyString" but you don't have a %s in the log message, so its not used. Need to fix that. Other than that, this is ready to be merged once that is fixed. So change this line to:

````
log.debugf("Hawkular Metrics is still starting: %s", bodyString);
````

While you are here, probably also good to add a colon to the previous debug log message to separate the static message with the value of bodyString - line 916:

````
log.debugf("Hawkular Metrics is ready: %s", bodyString);
````